.circleci/config.yml

```diff
@@ -366,6 +366,27 @@ jobs:
           docker login --password-stdin -u $DOCKER_USER <<<$DOCKER_PASS
           docker push tendermint/gaia:$CIRCLE_TAG
 
+  reproducible_builds:
+    <<: *linux_defaults
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          name: Build gaia
+          no_output_timeout: 20m
+          command: |
+            sudo apt-get install -y ruby
+            for os in darwin linux windows; do
+              bash ./cmd/gaia/contrib/gitian-build.sh ${os} `pwd`
+              cp gitian-build-${os}/result/gaia-${os}-res.yml .
+              rm -rf gitian-build-${os}/
+              echo "Build report for ${os}"
+              cat gaia-${os}-res.yml
+            done
+
```

Comment:
```
      - store_artifacts:
          path: /tmp/workspace/gitian-build-linux/result
      - store_artifacts:
          path: /tmp/workspace/gitian-build-windows/result
      - store_artifacts:
          path: /tmp/workspace/gitian-build-darwin/result
```

Reply:
sorry, wrong paths, I'm executing the job with ssh to check the correct ones

Reply:
Shouldn't we push the reports to something like S3 instead?

Reply:
I like that @mircea-c!

Reply:
wow I haven't seen the messages earlier, should be even better, we will do a follow up PR to move the artifacts from here to S3 or in this one?

Reply:
@sabau let's stick to this one